plaid/liabilities.go

```diff
@@ -0,0 +1,130 @@
+	ExpectedPayoffDate         string                     `json:"expected_payoff_date"`
+	Guarantor                  string                     `json:"guarantor"`
+	InterestRatePercentage     float64                    `json:"interest_rate_percentage"`
+	IsOverdue                  bool                       `json:"is_overdue"`
+	LastPaymentAmount          float64                    `json:"last_payment_amount"`
+	LastPaymentDate            string                     `json:"last_payment_date"`
+	LastStatementBalance       float64                    `json:"last_statement_balance"`
+	LastStatementIssueDate     string                     `json:"last_statement_issue_date"`
+	LoanName                   string                     `json:"loan_name"`
+	LoanStatus                 StudentLoanStatus          `json:"loan_status"`
+	MinimumPaymentAmount       float64                    `json:"minimum_payment_amount"`
+	NextPaymentDueDate         string                     `json:"next_payment_due_date"`
+	OriginationDate            string                     `json:"origination_date"`
+	OriginationPrincipalAmount float64                    `json:"origination_principal_amount"`
+	OutstandingInterestAmount  float64                    `json:"outstanding_interest_amount"`
+	PaymentReferenceNumber     string                     `json:"payment_reference_number"`
+	PSLFStatus                 PSLFStatus                 `json:"pslf_status"`
+	RepaymentPlan              StudentLoanRepaymentPlan   `json:"repayment_plan"`
+	SequenceNumber             string                     `json:"sequence_number"`
+	ServicerAddress            StudentLoanServicerAddress `json:"servicer_address"`
+	YTDInterestPaid            float64                    `json:"ytd_interest_paid"`
+	YTDPrincipalPaid           float64                    `json:"ytd_principal_paid"`
+}
+
+// PSLFStatus contains information about the student's eligibility in the
+// Public Service Loan Forgiveness program.
+type PSLFStatus struct {
+	EstimatedEligibilityDate string `json:"estimated_eligibility_date"`
+	PaymentsMade             int64  `json:"payments_made"`
+	PaymentsRemaining        int64  `json:"payments_remaining"`
+}
+
+// StudentLoanServicerAddress is the address of the servicer.
+type StudentLoanServicerAddress struct {
+	City       string `json:"city"`
+	Country    string `json:"country"`
+	PostalCode string `json:"postal_code"`
+	Region     string `json:"region"`
+	Street     string `json:"street"`
+}
+
+// StudentLoanStatus contains details about the status of the student loan.
+type StudentLoanStatus struct {
+	Type    string `json:"type"`
+	EndDate string `json:"end_date"`
+}
+
+// StudentLoanRepaymentPlan contains details about the repayment plan of the
+// loan.
+type StudentLoanRepaymentPlan struct {
+	Type        string `json:"type"`
+	Description string `json:"description"`
+}
+
+type getLiabilitiesRequestOptions struct {
```

Comment:
nit: should this be exported with `GetL...` so developers can reference this struct?

Reply:
eg: https://github.com/plaid/plaid-go/blob/master/plaid/investment_transactions.go#L34